packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -580,12 +592,12 @@ function MapAnalysisImpl(props: ImplProps) {
       children: [
         {
           type: 'subheading',
-          labelText: 'Single Variable Maps',
+          labelText: MapSideNavItemLabels.SingleVariableMaps,
           children: [
             {
               type: 'item',
               // concatenating the parent and subMenu labels creates a unique ID
-              id: MapSideNavItemLabels.ConfigureMap + MarkerTypeLabels.pie,
+              id: 'single-variable-pie',
```

Comment:
nit: we can remove the preceding comment re: concatenating labels

Reply:
Oh my, of course!